### DISCUSSION TITLE
StringGrid don't work

### DISCUSSION THREAD

**Discussion by JohnP61** [Embarcadero/DelphiFMX4Python#41] [Q&A]
I wanted to try "StringGrid" and got this error. Other components are working.
Am I doing something wrong?

![Sin título](https://user-images.githubusercontent.com/26121126/190522235-317d6a34-141d-4ce4-9896-bccc938627e4.png)

from delphifmx import *

class TodoApp(Form):

    def __init__(self, Owner):
        self.Caption = "Test grid"
        self.SetBounds(100, 100, 700, 500)

        self.grid = StringGrid(self)
        self.grid.Parent = self

def main():
    Application.Initialize()
    Application.Title = "TODO App"
    app = TodoApp(Application)
    Application.MainForm = app
    app.Show()
    Application.Run()
    app.Destroy()

if __name__ == '__main__':
    main()

**Comment by lmbelo** [comment]
Hello @JohnP61,

there is an [issue](https://github.com/Embarcadero/DelphiFMX4Python/issues/37) addressed to this error.

I will solve it as soon as possible.

**Comment by lmbelo** [comment]
That issue was fixed. Try it out :D

![Screenshot 2023-03-07 at 17 57 09](https://user-images.githubusercontent.com/8376898/223550878-4516b27b-fc33-4c65-a35c-13432e7bf71d.png)
